### ISSUE TITLE
Multiple cover specs found

### ISSUE THREAD

**Issue by sergeyu** [rebar/rebar#480]
I've a question. 
I'm running common tests using rebar:
```
rebar -v 1 ct skip_deps=true
```
rebar.conf contains:
```
{ct_extra_params, "-ct_hooks cth_surefire -logdir ct_result -cover test/cover.spec"}.
```
I see that rebar skips running ct in "deps", but still looking for "cover.spec" inside.
So I get an error:
``
ERROR: Multiple cover specs found
``

Could you please advise, how it can be resolved ?

**Comment by lrascao** [comment]
This is most likely already fixed, similar to #525, could you please check with rebar 2.6.0 is this still happens?

**Comment by lrascao** [comment]
Closing, please do re-open if issue still occurs
